tests/ui/consts/const_transmute_type_id2.rs

```diff
@@ -0,0 +1,14 @@
+//@ stderr-per-bitwidth
```

Comment:
Can we use normalization instead of two stderr files? That's much nicer for `--bless`.

Reply:
Yea sorry, wanted to wait for perf to start but left the computer before that happened

Reply:
Pushed